config/config.exs

```diff
@@ -168,8 +169,8 @@ config :trento, :jwt_authentication,
   issuer: "https://github.com/trento-project/web",
   app_audience: "trento_app",
   api_key_audience: "trento_api_key",
-  # Seconds, 10 minutes
-  access_token_expiration: 600,
+  # Seconds, 3 minutes
+  access_token_expiration: 180,
```

Comment:
Just curious why make it shorter ?

Reply:
In the new context where we have users, and we can remove/lock them, this deleting/locking takes effect immediately in the database, but as they authentication jwt token is stateless, you cannot change it.
So when you lock a user, the obtained access key would still be valid.
Reducing this time limits this "grace period" of 10 minutes to 3, so "not wanted users" validity time is reduced.
It is just a safety measure.